src/java.base/share/classes/java/lang/StringCoding.java

```diff
@@ -35,15 +35,18 @@ class StringCoding {
 
     private StringCoding() { }
 
-    public static int countGreaterThanZero(String s) {
+    /**
+     * Count the number of String::value leading non-negatives in the range.
+     */
+    public static int countNonNegatives(String s) {
```

Comment:
No, this is wrong, `NonNegatives` implies zeroes. I meant that `countPositives` should be renamed `countNonNegatives` in a future enhancement to more precisely reflect what it does. That's finicky since it's intrinsified and requires coordination with runtime code, though.